test/parallel/test-print-GH-23191.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const child = require('child_process');
+
+if (!common.isWindows) {
+  common.skip('This test is specific to Windows to test winapi_strerror');
+}
+
+// Ref: https://github.com/nodejs/node/issues/23191
+// This test is specific to Windows.
+
+const cp = child.spawn('pwd');
+
+cp.on('exit', common.mustCall(function(code) {
```

Comment:
```suggestion
cp.on('exit', common.mustCall(function() {
```

Or add an assertion for it.

Reply:
Thank you. Fixed it.